### PR TITLE
Mbs 8163 Update Hugo

### PR DESCRIPTION
**Pull request by eugene-krivobokov**

### CODE REVIEW

File: docs/Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ RUN apk update \
     # libc libraries are used to overcome https://github.com/gohugoio/hugo/issues/3382 😕
     libc6-compat libstdc++
 
-ARG HUGO_VERSION=0.64.1
+ARG HUGO_VERSION=0.69.2
```
Comment (eugene-krivobokov) [inline docs/Dockerfile:21]: https://github.com/gohugoio/hugo/releases

File: docs/Dockerfile [hunk 2]
```diff
@@ -1,12 +1,12 @@
 FROM ruby:2.6.5-alpine3.11 AS linter
 # mdl: https://github.com/markdownlint/markdownlint
-RUN gem install mdl -v 0.8.0
+RUN gem install mdl -v 0.9.0
```
Comment (eugene-krivobokov) [inline docs/Dockerfile:3]: https://github.com/markdownlint/markdownlint/blob/master/CHANGELOG.md

File: docs/content/menu/index.md [hunk 3]
```diff
@@ -68,4 +68,3 @@ headless: true
     - [QApps]({{< ref "/docs/cd/QApps.md" >}})
 - [Infrastructure (Github project)]({{< ref "/docs/Infrastructure.md" >}})
 - [Внешние библиотеки]({{< ref "/docs/ExternalLibraryChangeProcess.md" >}})
-- [Blog]({{< relref "/posts" >}})
```
Comment (eugene-krivobokov) [inline docs/content/menu/index.md:71]: Now it is added automatically

File: docs/themes/book/i18n/en.yaml [hunk 4]
```diff
@@ -12,3 +12,5 @@
 
 - id: bookSearchConfig
   translation: '{ cache: true, split: /[^a-zа-яё0-9\w]/gi }'
```
Comment (eugene-krivobokov) [inline docs/themes/book/i18n/en.yaml:14]: This is our workaround for pages with mixed language.

File: docs/layouts/shortcodes/mermaid.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{{ if not (.Page.Scratch.Get "mermaid") }}
+<!-- Include mermaid only first time -->
+<!--
+======
+This is a copy of https://github.com/alex-shpak/hugo-book/blob/master/layouts/shortcodes/mermaid.html
```
Comment (eugene-krivobokov) [inline docs/layouts/shortcodes/mermaid.html:5]: I've moved all our custom shortcodes to the root to keep theme as close to the original as possible.

File: docs/layouts/shortcodes/mermaid.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+Rationale for a custom implementation:
+- Be able to update it independently
+- Use CDN instead of inlining megabytes of js in repository
+======
+-->
+<script src="https://unpkg.com/mermaid@8.5.0/dist/mermaid.min.js" type="text/javascript" async></script>
```
Comment (eugene-krivobokov) [inline docs/layouts/shortcodes/mermaid.html:11]: https://github.com/mermaid-js/mermaid/releases

Supported a new type of UML diagram: ER
https://mermaid-js.github.io/mermaid/#/entityRelationshipDiagram

File: docs/themes/book/i18n/en.yaml (hunk 4 above)
Comment (eugene-krivobokov) [reply docs/themes/book/i18n/en.yaml:14]: I've found a solution without this hack, but it requires to change pages structure.
I'll make a proposal. Separate languages have some drawbacks.

Comment (int02h) [reply docs/themes/book/i18n/en.yaml:14]: Do we need to keep .yaml files for all languages we do not support?

File: docs/themes/book/layouts/partials/docs/html-head.html [hunk 7]
```diff
@@ -1,10 +1,11 @@
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<meta name="description" content="{{ partial "docs/title" . }}">
+<meta name="description" content="{{ default .Summary .Description }}">
+
 {{- template "_internal/opengraph.html" . -}}
 
 <title>{{ partial "docs/title" . }} | {{ .Site.Title -}}</title>
-<link rel="icon" href="{{ "favicon.ico" | relURL }}" type="image/x-icon">
+<link rel="icon" href="{{ "favicon.png" | relURL }}" type="image/x-icon">
```
Comment (int02h) [inline docs/themes/book/layouts/partials/docs/html-head.html:8]: What's wrong with .ico?

Comment (eugene-krivobokov) [reply docs/themes/book/layouts/partials/docs/html-head.html:8]: Nothing wrong. It was a [custom change](https://github.com/avito-tech/avito-android/pull/304/files) in the theme.
Any customization increases the price of updates. I've decided to fit the default one.

File: docs/themes/book/i18n/en.yaml (hunk 4 above)
Comment (eugene-krivobokov) [reply docs/themes/book/i18n/en.yaml:14]: My motivation was to reduce the diff in theme updates. 
These configs are cheap and don't work until you enable them in the config.

**Review by dsvoronin — APPROVED** [review]

**Pull request merged**